### ISSUE TITLE
t/04_methanol.t may fail (different sort order)

### ISSUE THREAD

**Issue by eserte** [merkys/Graph-Nauty#2]
On some of my smoker systems I see the following test failure:
```
#   Failed test at t/04_methanol.t line 29.
#          got: '[C][HB,HC,HA][HO][O]'
#     expected: '[C][HA,HB,HC][HO][O]'
# Looks like you failed 1 test of 2.
t/04_methanol.t .. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/2 subtests 
```
Maybe it's caused by hash randomization, or another source of randomness.

**Comment by merkys** [comment]
Thanks for reporting this. I suspect that `Graph::neighbour()` returns items in nondeterministic order. I will attempt to fix for the next release.

**Comment by merkys** [comment]
I have addressed this issue in dd605af626e853dffd46a2c3546b751934f3e90b. Please let me know if this resurfaces.

**Comment by merkys** [comment]
Closing the issue for now, please reopen if persists.
